src/Cli/dotnet/OptionForwardingExtensions.cs

```diff
@@ -18,6 +18,25 @@ public static class OptionForwardingExtensions
 
         public static ForwardedOption<T> ForwardAsSingle<T>(this ForwardedOption<T> option, Func<T, string> format) => option.SetForwardingFunction(format);
 
+        public static ForwardedOption<string> ForwardAsOutputPath(this ForwardedOption<string> option, string outputPropertyName, bool surroundWithDoubleQuotes = false)
```

Comment:
I think we should add a docstring that explains the `surroundWithDoubleQuotes` parameter, maybe just noting it's intended for `dotnet test` to preserve old behavior and remove the comment.